app/src/internalDebug/java/com/rockthevote/grommet/ui/debug/DebugDrawerLayout.java

```diff
@@ -489,6 +489,7 @@ public void setDrawerLockMode(@LockMode int lockMode, @EdgeGravity int edgeGravi
    * @see #LOCK_MODE_LOCKED_CLOSED
    * @see #LOCK_MODE_LOCKED_OPEN
    */
+  @SuppressLint("WrongConstant")
```

Comment:
The call to `setDrawerLockMode` (the one at the end of this function, it is overloaded) requires `@EdgeGravity`, but `gravity` is derived dynamically so can't be validated by the annotation.

Reply:
it's debug stuff anyway